client/src/main/java/org/apache/rocketmq/client/producer/MQProducer.java

```diff
@@ -70,12 +66,10 @@ SendResult send(final Message msg, final MessageQueueSelector selector, final Ob
         InterruptedException;
 
     void send(final Message msg, final MessageQueueSelector selector, final Object arg,
```

Comment:
How do we keep backward compatibility?

Reply:
Yeah, keep backward compatibility is very important and it will fixed in the next commit.